src/components/styles/StyledTabs.tsx

```diff
@@ -4,7 +4,7 @@ import { size } from "constants/tokens";
 import { TabsType } from "types/leafygreen";
 
 export const StyledTabs = styled<TabsType>(Tabs)`
-  [role="tablist"] {
+  > div > [role="tablist"] {
```

Comment:
Just a question: what's the meaning of this change? is it more correct?

Reply:
Segmented Control also has `role="tablist"`, so this rule was causing funky styling in those components. Using the direct child selectors just allows us to be more specific and only apply a margin directly below the tab component.